samples/jwt/boards/nrf9280pdk_nrf9280_cpurad.conf

```diff
@@ -0,0 +1 @@
+CONFIG_DCACHE=n
```

Comment:
Why are there Kconfig and DTS overlays for the radio core? I assume those are not needed by the sample.